liquibase-core/src/main/java/liquibase/util/TableOutput.java

```diff
@@ -19,7 +20,7 @@ public class TableOutput {
      * @param writer                   Writer to use for output
      *
      */
-    public static void formatOutput(List<List<String>> table, int[] maxWidths, boolean leftJustifiedRows, Writer writer) throws LiquibaseException {
+    public static void formatOutput(List<List<String>> table, List<Integer> maxWidths, boolean leftJustifiedRows, Writer writer) throws LiquibaseException {
```

Comment:
This (and the below version) is an API breaking change. Is there a compelling reason for that?

Reply:
There is not. I will overload this method so it's not a breaking change.